srcpkgs/ruby-build/template

```diff
@@ -1,14 +1,14 @@
 # Template file for 'ruby-build'
 pkgname=ruby-build
-version=20221124
+version=20240319
 revision=1
 depends="bash"
 short_desc="Compile and install Ruby"
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="Chris Saia <chris@todfox.net>"
```

Comment:
please don't adopt, we prefer adoptions come from contributors with a history of contributions to ensure packages get the most care possible